packages/nimble-components/src/table/specs/table-column-specs/table-column-elapsed-time-hld.md

```diff
@@ -0,0 +1,177 @@
+
+```ts
+const result = [];
+const hours = Math.floor(value / 3600);
+if (hours) {
+    const hoursFormatter = new Intl.NumberFormat(this.lang, { style: 'unit', unit: 'hour' });
+    const formattedHours = hoursFormatter.format(hours);
+    result.push(formattedHours);
+}
+... // do similar work for minutes and seconds
+```
+
+Other implementation details will conform to existing patterns all other table columns handle, such as `CellView` and `GroupHeaderView` implementations.
+
+## Alternative Implementations / Designs
+
+### Limited configuration
+
+_*Props/Attrs*_
+
+-   `field-name`: string
+-   `style`: `"short" | "narrow" | "long"`
+
+This would offer clients to display elapsed time in other forms, but not to intermix formats between the various parts.
+
+Examples:
+
+-   "short": "1 hr, 30 min, 2 sec"
+-   "narrow": "1h, 30m, 2s"
+-   "long": "1 hour, 30 minutes, 20 seconds"
+
+Pros:
+
+-   Offers a little more configuration
+-   Not much extra cost to provide more display flexibility
+-   Aligns with the `Intl.DurationFormat` API that is in development
+
+Cons:
+
+-   No immediate need identified that would leverage the flexibility
+-   Risk of introducing API that we ultimately would prefer not to use?
+
+### Full configuration
+
+This option would be more akin to the [`TableColumnDateText` API](https://github.com/ni/nimble/blob/990e415d40fa3abe36414c1736b2133f7cd0cd03/packages/nimble-components/src/table-column/date-text/index.ts#L57C2-L57C2), where we would mirror the API surface of the [`options` constructor parameter](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/DurationFormat/DurationFormat#parameters) of the `Intl.DurationFormat` object.
+
+Recommendation with this approach (if we decided to use this initially) would be to not support the `"digital"` `style` mode.
+
+Pros:
+
+-   Offers greatest set of flexibility for clients including the ability to display 'days' and 'years'.
+
+Cons:
+
+-   Increases implementation cost
+-   Higher risk of API ultimately not aligning with what we eventually want (particularly if `Intl.DurationFormat` is never formally adopted).
+
+Neither of the above options seem critical to implement initially, as we can always add features as needed.
+
+### States
+
+N/A
+
+### Accessibility
+
+### Mobile
+
+N/A
+
+### Globalization
+
+#### Localization
+
+Localization will be enabled through the `lang` setting provided by the `nimble-theme-provider` or root `<html>` element, which will then be passed as the `locale` to the JS native APIs (e.g. `Intl.NumberFormat`).
+
+### Security
+
+N/A
+
+### Performance
+
+N/A
+
+### Dependencies
+
+None
+
+### Test Plan
+
+Standard unit testing. Test cases of interest:
+
+-   unusual number values (NaN, -∞, ∞, negative values)
```

Comment:
I'm not sure if it belongs in the "Test Plan" section or some other section above, but I think it would be good to define the behavior for these "unusual number values". Will they all be rendered (with or without units)? Will some/all of them just display blank cells?